src/Routers/student_getall.js

```diff
@@ -0,0 +1,22 @@
+// this is the API is to give full Access to the admin 
+// here We are getting all the data by just callong the localhost:{port}/student_all
+module.exports = function (app, db) {
+    app.get("/student_all", (req, res) => {
```

Comment:
Is it just for testing during development ?
As a GET method is not secure and we're dumping complete table without verifying Admin's credentials .